trigger/version.go

```diff
@@ -193,7 +193,23 @@ func (t *versionTriggers) versionGithubCheckOutcome(sub *event.Subscription) (*n
 }
 
 func (t *versionTriggers) versionFailure(sub *event.Subscription) (*notification.Notification, error) {
-	if t.data.Status != evergreen.VersionFailed {
+	versionTasks, err := task.FindAll(task.ByVersion(t.version.Id))
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting tasks in version")
+	}
+	skipNotification := false
+	// If the only failures in the version come from aborted tasks then we skip sending a notification
+	for _, versionTask := range versionTasks {
+		if versionTask.Status == evergreen.TaskFailed {
+			if versionTask.Aborted {
+				skipNotification = true
```

Comment:
We don't need to store a skipNotification variable; we can just return here immediately if we hit this case

Reply:
I think we need this variable to ensure all failures are aborted tasks.  If we find a task that is failed and not aborted, then a notification may still need to be sent.